internal/service/rds/cluster.go

```diff
@@ -227,6 +227,12 @@ func ResourceCluster() *schema.Resource {
 								"RollbackCapacityChange",
 							}, false),
 						},
+						"seconds_before_timeout": {
+							Type:         schema.TypeInt,
+							Optional:     true,
+							Default:      300,
+							ValidateFunc: validation.IntBetween(10, 600),
```

Comment:
I believe the supported configurations are between 60 and 600 according to the API documentation.

Reply:
I have corrected this